cypress/support/commands.js

```diff
@@ -508,45 +562,34 @@ Cypress.Commands.add('declareDeathApplicationWithMaximumInput', () => {
   cy.enterDeathMaximumInput()
   // PREVIEW
   cy.submitApplication()
-  cy.get('#row_0 #submitted0').should('exist')
+  //cy.get('#row_0 #submitted0').should('exist')
```

Comment:
@azantor64 by commenting this out you are not testing that the registration successfully submitted and appeared in the workqueue